modules/ROOT/pages/java-support.adoc

```diff
@@ -33,6 +33,25 @@ The maximum supported version for running your applications is Java 17 for Mule
 
 For more information about Mule release cadences, see xref:release-notes::mule-runtime/lts-edge-release-cadence.adoc[].
 
+== Steps for Upgrade 
+
+Follow these steps to upgrade to Java 17:
+
+. Learn about <<application-performance, application performance>> when upgrading to Java 17. 
+. Conduct an analysis of your current setup to pinpoint compatibility issues or deprecated elements.
+. Create a comprehensive upgrade plan, including testing, rollback strategies, and post-upgrade checks.
+. Schedule downtime and inform stakeholders to minimize business disruption.
+. Implement the upgrade:
```

Comment:
no general upgrade documentation we can point to as a basis?

Reply:
This page is the general upgrade doc for all things Java 17. In each step, I link to the relevant docs.